cmd/gh-action-integration-generator/main.go

```diff
@@ -125,61 +26,44 @@ func findTests() []string {
 		"--no-heading",
 	}
 
-	log.Printf("executing: %s %s", rgBin, strings.Join(args, " "))
-
-	ripgrep := exec.Command(
-		rgBin,
-		args...,
-	)
-
-	result, err := ripgrep.CombinedOutput()
+	cmd := exec.Command(rgBin, args...)
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err = cmd.Run()
 	if err != nil {
-		log.Printf("out: %s", result)
-		log.Fatalf("failed to run ripgrep: %s", err)
+		log.Fatalf("failed to run command: %s", err)
 	}
 
-	tests := strings.Split(string(result), "\n")
-	tests = tests[:len(tests)-1]
-
+	tests := strings.Split(strings.TrimSpace(out.String()), "\n")
 	return tests
 }
 
-func main() {
-	type testConfig struct {
-		Name     string
-		Test     string
-		Postgres bool
-	}
-
-	tests := findTests()
+func updateYAML(tests []string) {
+	testsForYq := fmt.Sprintf("[%s]", strings.Join(tests, ", "))
 
-	removeTests()
-
-	for _, test := range tests {
-		for _, postgres := range []bool{false, true} {
-			log.Printf("generating workflow for %s", test)
-
-			name := test
-			if postgres {
-				name = test + "-postgres"
-			}
+	yqCommand := fmt.Sprintf(
+		"yq eval '.jobs.integration-test.strategy.matrix.test = %s' ../../.github/workflows/test-integration.yaml -i",
```

Comment:
can you add `yq` to the flake.nix file so its automatically installed for the dev env? that way we should not run into an issue with needing it.

Reply:
done

Reply:
Little sidenote, is there any particular reason why `gofumpt` is not part of the devdeps in the nix shel?

Reply:
No it should be, I must have missed it, that can be done in a followup.

Currently we are lacking a bit on jobs to verify the formatting, so a job similar to the "is the tests generated" would potentially be beneficial for golines + gofumpt.